log2ram.conf

```diff
@@ -14,4 +14,20 @@ USE_RSYNC=false
 
 # If there are some errors with available RAM space, a system mail will be send
 # Change it to false and you will have only a log if there is no place on RAM anymore.
-MAIL=true
\ No newline at end of file
+MAIL=true
+
+# **************** Zram backing conf  *************************************************
+
+# ZL2R Zram Log 2 Ram enables a zram drive when ZL2R=true ZL2R=false is mem only tmpfs
+ZL2R=false
+# COMP_ALG this is any compression algorithm listed in /proc/crypto
+# lz4 is fastest with lightest load but deflate (zlib) and Zstandard (zstd) give far better compression ratios
+# lzo is very close to lz4 and may with some binaries have better optimisation
+# COMP_ALG=lz4 for speed or Zstd for compression, lzo or zlib if optimisation or availabilty is a problem
+COMP_ALG=lz4
+# LOG_DISK_SIZE is the uncompressed disk size. Note zram uses about 0.1% of the size of the disk when not in use
+# LOG_DISK_SIZE is expected compression ratio of alg chosen multiplied by log SIZE
+# lzo/lz4=2.1:1 compression ratio zlib=2.7:1 zstandard=2.9:1
+# Really a guestimate of a bit bigger than compression ratio whilst minimising 0.1% mem usage of disk size
+LOG_DISK_SIZE=100M
```

Comment:
Why having 2 different log size variables ? Confusing, especially when you don't know the compression ratio.

Reply:
Its due to the way zram was developed as mem_limit didn't exist at first then someone correctly pointed out that using drive_size (uncompressed size) is a virtual size and means absolutely nothing and for sys_admins who must have control of system mem_limit means if a page is uncompressible or near setting page size means assuming this condition and so negating any possible compression as you are forced to pick the worst scenario.
mem_limit was tacked on afterwards to fix this as even though drive size is the main method of control its use is near pointless.
In fact you have to set drive_size as it creates a virtual memory buffer size that actually has overhead and it is confusing because apart wasting 0.1% of the memory of the drive size you make it does absolutely nothing but restrict max compression ratio that you might get with empty hollow pages.
Size is exactly as it was and translates to the max amount of memory that will be used by zram and the defaults even with LZ4 are extremely conservative.
If you read the README.md you will see a table of compression algorithms stating minimum compression ration.
With text logs and text in general compression ratios are vastly greater than that but it can not be hard set as really it means nothing apart from to limit the 0.1% mem usage of this size even when empty that will also likely set the max compression ratio and it was the main method of control until someone pointed out it means absolutely nothing as its a virtual size and they created real size mem_limit control to fix this.
Debian zram_config hasn't been changed from before mem_limit was added and still uses virtual size alone.
The Armbian guys do something a little stranger as they set default mem_limit & drive_size to the same as as it gets full you have essentially the same as a normal ram drive but with compression overhead.
So basically the only limit on LOG_DISK_SIZE is that it will waste  0.1% of mem even when empty and yeah you can set this to any size as mem_limit does do the control and max compression ratio is totally dependent on input.
The 250% default expect the lowest compression rates of LZO/LZ4 is extremely conservative in terms of 0.1% mem usage and uses approx 100M*0.1%= approx 100k to set up the virtual memory buffer.
In use with text logs this is likely to be much more but its dependent on input, has an overhead and is one of those how long is a piece of string questions.
With LZO/LZ4 generally I still go conservative and drop the size (mem_limit) to 50% (20M) and have a disk_size of (350% - 400%)
With Deflate or Zstandard you could prob drop to (10m - 15m) and have a disk size of (450% - 600%)
Without ever hitting the (10m - 15m) hard setting of mem_limit.

Zram is a virtual device so it has to manners of control one drive_size that purely minimises 0.1% mem wastage of this setting even when empty.
Then the crucial sysadmin control of maximum memory to be used by Zram of mem_limit.

Confusing yeah but its really just a matter of choice and because of that definitely should not be hard set or missing.